docs/source/_templates/class.rst

```diff
@@ -0,0 +1,32 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+   :members:
+   :show-inheritance:
+   :inherited-members:
```

Comment:
It looks like these directives are the main addition being made to the default template here. Do you really think it's worthwhile? I'm weakly opposed since so few people actually know enough about how autosummary works to know what this is doing, but I'm OK with it if you really think it's worth it.

Reply:
I feel the same as Vyas -- but I didn't think this was worth changing. If you're happy, I think it's good enough, and we should expend effort on the next steps.